Source/WebKit/UIProcess/Cocoa/SystemPreviewControllerCocoa.mm

```diff
@@ -361,7 +361,7 @@ - (UIImage *)previewController:(QLPreviewController *)controller transitionImage
 {
     auto uuid = UUID::parseVersion4(String(documentID));
     ASSERT(uuid);
-    if (!uuid)
+    if (!uuid || !uuid->isValid())
         return;
```

Comment:
If we change UUID::parse(), we might want to look at WebsiteDataStore::fetchAllDataStoreIdentifiers.

Reply:
Updated patch does not change this behavior.